### DISCUSSION TITLE
Microsoft.Extensions.Caching.StackExchangeRedis version 5

### DISCUSSION THREAD

**Discussion by arisewanggithub** [dotnet/aspnetcore#29366] [Q&A] [answered]
Is Microsoft.Extensions.Caching.StackExchangeRedis package version 5 only for .net 5? Or .net core 3.1 apps can also use it?
And can .net 5 apps uses the 3.1 version of this package?

**Comment by Pilchie** [comment, marked as answer]
Generally we recommend matching the major.minor version to the .NET Core stack that you are using for all of our Microsoft.Extensions packages.
